cumulusci.yml

```diff
@@ -35,6 +35,11 @@ tasks:
               - cumulusci/robotframework/pageobjects/BasePageObjects.py
 
 flows:
+    qa_org:
+        steps:
+            2:
+                flow: None
```

Comment:
Do we need this? CumulusCI doesn't have any tests which depend on things set up by the qa_org flow, does it?

Reply:
Well I want to test features that depend on config-ing an org and the org-configuring flows crash on main due to trying to deploy non-existent metadata.

Reply:
I could make a new flow, but then the examples will be less "realistic".

Reply:
The deploy_unmanaged flow is supposed to silently do nothing if there's not actually a package directory, so this feels like the test may actually be showing a bug that you're covering up here.

I have mixed feelings about adding this when we don't actually have tests that need it. On the one hand, we're more likely to add tests like that if the infrastructure already exists. On the other hand, it's hard to be confident that we're building the right infrastructure without a concrete use case.